app/models/katello/content_view.rb

```diff
@@ -108,7 +108,7 @@ def self.published_with_repositories(root_repository)
     end
 
     def self.in_organization(org)
-      where(organization_id: org.id)
+      where(organization_id: org.id) unless org.nil?
     end
```

Comment:
Probably don't need this anymore..were you seeing an error in the UI with "Any Organization"  selected?

Reply:
without it there was an error in the organization creation tests because the create rabl extends the show rabl